src/test/java/TestMarsRover.java

```diff
@@ -0,0 +1,135 @@
+import org.example.MarsRover;
+import org.example.Rover;
+import org.junit.jupiter.api.Assertions;
+import org.junit.Test;
+
+public class TestMarsRover {
+
+    @Test
+    public void testInitialPosition()
+    {
+        Rover rover = new MarsRover();
+        rover.initialRoverLocation(1,1,'N');
+        char[] commands = {'f','f','l','f','r'};
+        rover.roverCommand(commands);
+        rover.presentLocation();
+        rover.commandsToBeFollowed();
+    }
```

Comment:
what is asserted in this test

Reply:
This is the very first basic test when we were just checking weather the flow is working fine or not by printing the present location and what commands are given to the rover. hence no assertions.

Reply:
no assertion is no unit test. even sonar will complain. u can assert no exception is thrown.
https://rules.sonarsource.com/java/tag/junit/RSPEC-2699/.

Reply:
okay ! i will take care of it. Thanks.